.github/scripts/install_crds.sh

```diff
@@ -7,20 +7,20 @@
 # - Connection to a running k8s cluster (e.g. kind)
 
 install_appstudio_repo_crds () {
-    git clone https://github.com/redhat-appstudio/$1
-    pushd $1
+    git clone https://github.com/$1/$2
+    pushd $2
     kubectl apply -f config/crd/bases
     popd
 }
 
 if ! kubectl get crd snapshots.appstudio.redhat.com ; then
-    install_appstudio_repo_crds "application-api"
+    install_appstudio_repo_crds "redhat-appstudio" "application-api"
 fi
 
 if ! kubectl get crd releases.appstudio.redhat.com ; then
```

Comment:
I have no idea how this works right now, but I think this should be internalrequests.appstudio.redhat.com, no?